src/copy.ml

```diff
@@ -914,14 +914,8 @@ let copythreshold =
 let copyquoterem =
   Prefs.createBoolWithDefault "copyquoterem"
     ~category:(`Advanced `General)
-    "add quotes to remote file name for copyprog (true/false/default)"
-    ("When set to {\\tt true}, this flag causes Unison to add an extra layer "
-     ^ "of quotes to the remote path passed to the external copy program. "
-     ^ "This is needed by rsync, for example, which internally uses an ssh "
-     ^ "connection requiring an extra level of quoting for paths containing "
-     ^ "spaces. When this flag is set to {\\tt default}, extra quotes are "
-     ^ "added if the value of {\\tt copyprog} contains the string "
-     ^ "{\\tt rsync}.")
+    "ignored for backward compatibility"
```

Comment:
Instead of this, use  `Prefs.markRemoved`. Example: 7a28f01c0

(Alternative is to first use `~deprecated:true` for a couple of releases but in this case it may be ok to remove directly.)

Reply:
I've used `Prefs.markremoved`.